languages/common-lisp/exercises/concept/basic-lists/.docs/instructions.md

```diff
@@ -0,0 +1,49 @@
+Leslie the Lisp Alien needs to do some shopping. It is very important to have a shopping list. One needs to add things to it, remove things from it, and even look if something is on the list for instance.
+
+Of course simple pen and paper will not do for a Lisp Alien. "List" is most of the word "Lisp" even!. There must be some functions written to help keep track of the shopping.
+
+Can you help Leslie keep track of the shopping list?
+
+## 0. Making a new list
```

Comment:
Yes, 0th is an affectation of mine. But also I was, wrongly, thinking about how in markdown when you have numbered items that it doesn't matter the number you put into your markdown doc - it will be rendered by a correct sequence of numbers. (Of course that is not the case here...)

Reply:
Huh, you learn something new about markdown every day... That's pretty neat! Though I suppose they aren't expecting headers like these to be numbered